src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arr = [];
+  const clonedState = { ...state };
+
+  if (actions) {
```

Comment:
No need to check if `actions` are exists in this task

Reply:
If you remove this check, the tests fail.
![image](https://github.com/mate-academy/js_stateful-clones/assets/139628544/ccfaba83-aced-4c8b-8901-acb69bc093e3)

`stateful-clones@1.0.0 lint C:\projects\js_stateful-clones
> mate-scripts lint


> stateful-clones@1.0.0 test:only C:\projects\js_stateful-clones      
> mate-scripts test

 FAIL  src/transformStateWithClones.test.js
  ● Test suite failed to run       

    TypeError: Cannot read property 'forEach' of undefined

      11 |   const clonedState = { 
...state };
      12 | 
    > 13 |   actions.forEach(function(item) {
         |           ^
      14 |     if (item.type === 'addProperties') {
      15 |       for (const [key, value] of Object.entries(item.extraData)) {
      16 |         clonedState[key] = value;

      at transformStateWithClones (src/transformStateWithClones.js:13:11)
      at Object.<anonymous> (src/transformStateWithClones.js:39:1)    
      at Object.<anonymous> (src/transformStateWithClones.test.js:3:34)

Test Suites: 1 failed, 1 total     
Tests:       0 total
Snapshots:   0 total
Time:        3.581s
Ran all test suites matching /.\\/i.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! stateful-clones@1.0.0 test:only: `mate-scripts test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the stateful-clones@1.0.0 test:only script.        
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.       

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\oleks\AppData\Roaming\npm-cache\_logs\2023-10-28T14_35_10_545Z-debug.log
npm ERR! Test failed.  See above for more details.
PS C:\projects\js_stateful-clones> `